docs/resources/tunnel_config.md

```diff
@@ -9,8 +9,6 @@ description: |-
 
 Provides a Cloudflare Tunnel configuration resource.
 
-!> When you delete a tunnel configuration, the tunnel will be deleted. You need to make sure that the tunnel is not in use before deleting the configuration.
```

Comment:
unfortunately, the file you've edited is a generated artifact of the documentation process. to see how to make changes to the documentation, you're best to check out https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/contributing/registry-documentation.md